openshift/android-sdk-sample-pv.json

```diff
@@ -13,7 +13,7 @@
     ],
     "persistentVolumeReclaimPolicy": "Recycle",
     "hostPath": {
-      "path": "/tmp/build-farm"
+      "path": "/tmp/build-farm/android-sdk-linux"
```

Comment:
we could also add some examples which allow nfs and gluster storage support, [here](https://gist.github.com/laurafitzgerald/d7b329c896c7e93b2aa0f9d1e7aa9404) is a sample